src/vmm/src/cpu_config/x86_64/static_cpu_templates/t2a.rs

```diff
@@ -18,25 +19,55 @@ pub fn t2a() -> CustomCpuTemplate {
                 subleaf: 0x0,
                 flags: KvmCpuidFlags(0),
                 modifiers: vec![
+                    // EAX: Version Information
+                    // - Bits 03-00: Stepping (AMD APM) / Stepping ID (Intel SDM)
+                    // - Bits 07-04: BaseModel (AMD APM) / Model (Intel SDM)
+                    // - Bits 11-08: BaseFamily (AMD APM) / Family (Intel SDM)
+                    // - Bits 13-12: Reserved (AMD APM) / Processor Type (Intel SDM)
+                    // - Bits 19-16: ExtModel (AMD APM) / Extended Model ID (Intel SDM)
+                    // - Bits 27-20: ExtFamily (AMD APM) / Extended Family ID (Intel SDM)
                     CpuidRegisterModifier {
                         register: CpuidRegister::Eax,
                         bitmap: RegisterValueFilter {
-                            filter: 0b00001111111111110011111111111111,
-                            value: 0b00000000000000110000011011110010,
+                            filter: 0b0000_11111111_1111_00_11_1111_1111_1111,
+                            value: 0b0000_00000000_0011_00_00_0110_1111_0010,
                         },
                     },
+                    // ECX: Feature Information
+                    // - Bit 02: Reserved (AMD APM) / DTES64 (Intel SDM)
```

Comment:
Any particular reason to have a CPU template intended for AMD describing the Intel specification as well?

Reply:
T2CL and T2A is a pair of CPU templates that aims to provide instruction set-level feature parity on Intel and AMD. To provide information about why the bit is disabled, having both information would be helpful.

Reply:
I'm not sure feature parity is something we could guarantee. Even implying this is a stretch in my opinion. The templates shouldn't be used on platforms they weren't written for, and differentiation between the vendors should be expected. Having the comments on the templates is not a big deal though and could be informative.

Reply:
You're right. These information are added for our maintainability, not for customers. We have public documentation about their usage for customers: https://github.com/firecracker-microvm/firecracker/blob/main/docs/cpu_templates/cpu-templates.md